File: Rating (800)/A_Increase_or_Smash.go
package main

import (
	"fmt"
	"sort"
)

func main () {
	var testCase int;
	fmt.Scan(&testCase);

	for i:=0; i < testCase; i++ {
		var arrayLength int;
		fmt.Scan(&arrayLength)
		count := 1

		arr := make([]int, arrayLength)

		for j:=0; j<arrayLength; j++ {
			fmt.Scan(&arr[j])
		}

		sort.Ints(arr)

		for j:=1; j<arrayLength; j++ {
			if arr[j] != arr[j-1] {
				count++
			}
		}
		sum := (2*count) - 1
		fmt.Println(sum)
	}
}

File: Rating (800)/A_Increase_or_Smash.java
import java.util.*;
public class A_Increase_or_Smash {
    public static void main (String [] args) {
        Scanner sc = new Scanner(System.in);
        int testCase = sc.nextInt();

        while (testCase-- > 0) {
            int arryLength = sc.nextInt();
            int arr [] = new int[arryLength];
            int count = 1;

            for (int i=0; i<arryLength; i++) {
                arr[i] = sc.nextInt();
            }
            Arrays.sort(arr);

            for (int i=1; i<arr.length; i++) {
                if (arr[i] != arr[i-1]) {
                    count ++;
                    
                }
            } int sum = (2 * count)-1;
                System.out.println(sum);
        }
    }
}